backend/tests/utils/file_utils_test.py

```diff
@@ -1,34 +1,34 @@
-from io import BytesIO
-from typing import BinaryIO
 from unittest.mock import MagicMock
-from fastapi import HTTPException, UploadFile
-from fastapi.datastructures import Headers
+from fastapi import HTTPException
+
 import pytest
 
+from src.llm.llm import LLMFile
 from src.utils.file_utils import handle_file_upload
 
 
 def test_handle_file_upload_size():
     with pytest.raises(HTTPException) as err:
-        handle_file_upload(UploadFile(file=BinaryIO(), size=15*1024*1024))
+        large_file_content = b"x" * (15*1024*1024 + 1)
```

Comment:
I just checked and the `sys.getsizeof(large_file_content)` is `15,728,674` and that is higher than `15*1024*1024 + 1 = 15,728,641` 👍 happy with this. Maybe worth adding some of these numbers as comments for future reference